db/migrate/20240111192846_create_likes.rb

```diff
@@ -3,8 +3,8 @@ def change
     create_table :likes do |t|
       t.references :user, null: false, foreign_key: true
       t.references :post, null: false, foreign_key: true
-      t.datetime :created_at
-      t.datetime :updated_at
+      t.datetime :createdAt
+      t.datetime :updatedAt
```

Comment:
- [ ] **[OPTIONAL]** I would suggest you remove these columns. Besides, the fact they do not follow snake case rule for table columns, there is no use for these `updateAt` and `createdAt` columns. The `t.timestamps` takes care of the `updated_at` and `created_at` columns that we need.